docker/env/hydra.sh

```diff
@@ -88,6 +88,7 @@ function run_in_docker () {
         -e JOB_NAME="${JOB_NAME}" \
         -e BUILD_URL="${BUILD_URL}" \
         -e _SCT_BASE_DIR="${SCT_DIR}" \
+        -e RUN_BY_USER="${RUN_BY_USER}" \
```

Comment:
`-e RUN_BY_USER` is enough.

Reply:
Having it this way allows you to have whole docker command with all data in it, which is usefull if you want to reproduce it.